docs/README.md

```diff
@@ -0,0 +1,34 @@
+# Documentation Syncing with GitHub Actions
+
+## Overview
+
+This repository uses GitHub Actions to automatically sync documentation upon merging to the `main` branch. Changes in specific folders trigger actions that copy the contents to designated directories, overriding existing files and deleting files in the destination if they do not exist in the source.
+
+## Trigger Folders and Destinations
+
+The following folders trigger sync actions when they are changed:
+
+- `sso`
+- `snap`
+- `common`
+
+### Transfer Details
+
+- **sso**: Contents are copied to `[nuficonnect]/integration/instructions`.
+- **snap**: Contents are copied to `[snap]/integration/instructions`.
+- **common**: Contents are copied to both `[nuficonnect]/integration/common` and `[snap]/integration/common` to maintain relative paths and prepare for use as sub-pages on GitBook. This approach is necessary because referencing a single file on GitBook multiple times does not work (in case of subpages, however these are not desired at this moment).
```

Comment:
> referencing a single file on GitBook multiple times does not work (in case of subpages, however these are not desired at this moment)

I don't understand what "these" in this sentence means, I'd consider rewording the sentence